rllib2/algorithms/ppo/torch/unit_trainer.py

```diff
@@ -0,0 +1,37 @@
+
+from typing import Dict
+import torch
+from torch.optim import Optimizer, Adam
+
+from rllib2.data.sample_batch import SampleBatch
+from rllib2.core.torch.torch_unit_trainer import UnitTrainer
+from rllib2.core.torch.torch_rl_module import TorchRLModule
+
+
+class PPOTorchUnitTrainer(UnitTrainer):
```

Comment:
Should we just call these e.g. `PPOTorch`? That would simplify the nomenclature a bit AND make it very clear to the users that everything Torch specific that PPO "brings with it" (e.g. loss) should go into this class.

I do like "UnitTrainer" (now that we disambiguated and no longer have RLlib Trainers), but also ok with "TrainUnit" (shorter and kind of sounds better).